benefit/data/sheets/CaseEventToFields/CaseEventToFields.json

```diff
@@ -22,6 +22,30 @@
     "PageID": "1.0",
     "PageLabel": "Handle Further Evidence"
   },
+  {
+    "CaseEventID": "actionFurtherEvidence",
+    "CaseFieldID": "showPostponementDetailsPage",
+    "CaseTypeID": "Benefit",
+    "DisplayContext": "MANDATORY",
+    "LiveFrom": "21/08/2021",
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 1,
+    "PageID": "2.0",
+    "PageLabel": "Hearing request party",
+    "PageShowCondition": "showPostponementDetailsPage=\"Yes\"",
```

Comment:
Where is this set? I can't see any code in Tribunals or anywhere that sets it but maybe im looking in wrong place?

Reply:
Also, can you move this so it appears below all the things in page 1 as it's now in the middle of 1.0

Reply:
this flag is set in ActionFurtherEvidenceMidEventHandler
Moved to the end of the page.